nifi-registry-core/nifi-registry-framework/src/main/java/org/apache/nifi/registry/security/authorization/file/FileAccessPolicyProvider.java

```diff
@@ -215,6 +219,23 @@ public void onConfigured(AuthorizerConfigurationContext configurationContext) th
                 }
             }
 
+            PropertyValue identityGroupNameProp = configurationContext.getProperty(PROP_NIFI_IDENTITY_GROUP_NAME);
+            String identityGroupName = (identityGroupNameProp != null && identityGroupNameProp.isSet()) ? identityGroupNameProp.getValue() : null;
+            if (identityGroupName != null) {
```

Comment:
I haven't dug into the difference yet, but in NiFi if you have `<property name="Node Group"></property>`, it doesn't look for the group and continues on with normal start up, but in registry if you have `<property name="Identity Group Name"></property>` then it throws an exception:

`SecurityProviderCreationException: Authorizations node group '' could not be found`

We should probably try to have the same behavior here.